wallet-reference-server/src/main/resources/assets/stellar.toml

```diff
@@ -0,0 +1,5 @@
+ACCOUNTS = [ "GAJ3JPQH3ISAZBMRUPKQ6ZQLRXIROCKJKBIYMGPFUEETQEDGFAW4IQ5H" ]
```

Comment:
The location is confusing because this is not part of an asssets file.

Reply:
What do you mean?